### PR TITLE
Add string inside Donut chart

### PR DESCRIPTION
**Pull request by RajeshGoriga**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

Add string inside Donut chart 

#### Focus areas to test

Donut chart

#### Screenshot 

image

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/fluentui/pull/12435)

### CODE REVIEW

File: packages/charting/src/components/DonutChart/Arc/Arc.types.ts [hunk 1]
```diff
@@ -66,6 +66,11 @@ export interface IArcProps {
    * internal prop for href
    */
   href?: string;
+
+  /**
+   * props for inside donut value
+   */
+  insideDonutValue?: string | number;
```
Comment (aneeshack4) [inline packages/charting/src/components/DonutChart/Arc/Arc.types.ts]: nit: valueInsideDonut

File: change/@uifabric-charting-2020-03-26-19-19-23-user-v-satgar-PBI_4052037.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "type": "minor",
+  "comment": "Add string inside Donut chart",
```
Comment (aneeshack4) [inline change/@uifabric-charting-2020-03-26-19-19-23-user-v-satgar-PBI_4052037.json]: change to more consistent release note comment structure "DonutChart: Add prop to allow string inside Donut."

Comment (RajeshGoriga) [reply change/@uifabric-charting-2020-03-26-19-19-23-user-v-satgar-PBI_4052037.json]: Changed as suggested

File: packages/charting/src/components/DonutChart/Arc/Arc.types.ts (hunk 1 above)
Comment (RajeshGoriga) [reply packages/charting/src/components/DonutChart/Arc/Arc.types.ts]: Changed as suggested

**Pull request merged**